Userland/Libraries/LibWeb/Fetch/Infrastructure/HTTP/Responses.cpp

```diff
@@ -76,6 +77,25 @@ ErrorOr<Optional<AK::URL>> Response::location_url(Optional<String> const& reques
     return location;
 }
 
+// https://fetch.spec.whatwg.org/#concept-response-clone
+NonnullOwnPtr<Response> Response::clone() const
+{
+    // To clone a response response, run these steps:
+
+    // 1. If response is a filtered response, then return a new identical filtered response whose internal response is a clone of response’s internal response.
```

Comment:
Just forgot to implement it :sweat_smile:
Will mark as FIXME for now and try to implement tomorrow before merging this.

Reply:
I think this will require a change from making `Response` single-ownership to ref-counted, as filtered responses may be a view on an existing response (I think?) or a newly cloned one, which then has no owner.